apps/examples/nettest/nettest.c

```diff
@@ -214,6 +217,53 @@ static void show_usage(void)
 
 /* ------------------------------------------------------------ */
 /*                                                              */
+/* Send Broadcast example.                                      */
+/*                                                              */
+/* ------------------------------------------------------------ */
+void broadcast_send(uint32_t num_packets, uint32_t sleep_time)
+{
+	int sock;
+	int is_broadcast = 1;
+	int res = -1;
+	struct sockaddr_in dest_addr;
+	const char databuf[] = "broadcast msg from TizenRT";
+
+	sock = socket(AF_INET, SOCK_DGRAM, 0);
+	if (sock == -1) {
+		printf("[BRCLIENT] socket create fail err(%d)\n", errno);
+		return;
+	}
+
+	res = setsockopt(sock, SOL_SOCKET, SO_BROADCAST, &is_broadcast, sizeof(is_broadcast));
+	if (res == -1) {
+		printf("[BRCLIENT] set socket option fail err(%d)\n", errno);
+		goto return_with_close;
+	}
+
+	dest_addr.sin_family = AF_INET;
+	dest_addr.sin_port = htons(g_app_target_port);
+	dest_addr.sin_addr.s_addr = inet_addr(g_app_target_addr);
+	int i = 1;
+	for (; i <= num_packets; i++) {
+		res = sendto(sock, databuf, sizeof(databuf), 0, (struct sockaddr *)&dest_addr, sizeof(dest_addr));
+		if (res < 0) {
+			printf("[BRCLIENT] [ERR] sending datagram message (%d)", errno);
+			goto return_with_close;
+		}
+		printf("[BRCLIENT] sending broadcast message length (%d) number (%d)\n", sizeof(databuf), i);
```

Comment:
sendto() returns the total number of bytes sent.
Instead of 'sizeof(databuf)', why don't you use 'res' to print the total number of bytes sent?

Reply:
ok, that is better